src/components/tutorial-card/types.ts

```diff
@@ -0,0 +1,38 @@
+import { ProductSlug } from 'types/products'
+
+export interface TutorialCardProps {
+  /**
+   * A short description of the tutorial.
+   */
+  description: string
+
+  /**
+   * A string representation the duration of the tutorial, such as "10mins".
+   */
+  duration: string
+
+  /**
+   * Whether the tutorial has an embedded interactive lab.
+   */
+  hasInteractiveLab: boolean
+
+  /**
+   * Whether the tutorial has a video associated with it.
+   */
+  hasVideo: boolean
+
+  /**
+   * The title of the tutorial.
+   */
+  heading: string
+
+  /**
+   * A URL that links to the tutorial.
+   */
+  url: string
+
+  /**
+   * A list of product slugs, representing products used in the tutorial.
+   */
+  productsUsed: ProductSlug[]
```

Comment:
```suggestion
  productsUsed: ProductOption[]
```
There's a subtle difference but since the data is coming from the learn api I think we should use this.

Reply:
Thats in `lib/learn-client/types` I think

Reply:
✅ Updated to use `ProductOption`